frontend/ui/widget/configdialog.lua

```diff
@@ -932,20 +935,28 @@ function ConfigDialog:onConfigChoice(option_name, option_value)
     return true
 end
 
-function ConfigDialog:onConfigEvent(option_event, option_arg, when_applied_callback)
+function ConfigDialog:onConfigEvent(option_event, option_arg, when_applied_callback, verbosity)
+    Notification:setNotificationSource(2)
+    Notification:setNotificationVerbosity(verbosity or 0)
     self.ui:handleEvent(Event:new(option_event, option_arg, when_applied_callback))
+    Notification:setNotificationSource(0)
+    Notification:setNotificationVerbosity(0)
     return true
 end
 
-function ConfigDialog:onConfigEvents(option_events, arg_index)
+function ConfigDialog:onConfigEvents(option_events, arg_index, verbosity)
+    Notification:setNotificationSource(2)
```

Comment:
To avoid having `verbosity` and `more_verbosity` (depending if the event comes from the widget or from a NumberPicker), can't we have another source
```lua
Notification:setNotificationSource(Notification.SOURCE_BOTTOM_MENU)
Notification:setNotificationSource(Notification.SOURCE_BOTTOM_MENU_FINE_TUNING)
```
that would probably need to set elsewhere before calling the common onConfigEvents.